src/plugins/imagery/components/ImageryViewLayout.vue

```diff
@@ -1,3 +1,25 @@
+/*****************************************************************************
```

Comment:
This component is getting way too big. It's approaching 1000 LOC, so we need to break it up. My suggestion would be to move all of the "related telemetry" logic into a RelatedTelemetry class. This would encapsulate all of the logic for loading related telemetry objects, as well as subscribing to them and fetching latest data.

Reply:
Done! Updated test notes since they've changed.